runtime/calamari/src/fee.rs

```diff
@@ -40,16 +40,11 @@ pub struct WeightToFee;
 impl WeightToFeePolynomial for WeightToFee {
 	type Balance = Balance;
 	fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
-		// in Polkadot, extrinsic base weight (smallest non-zero weight) is mapped to 1/10 CENT:
-		// in Manta Parachain, we map to 1/10 of that, or 1/100 CENT
-		// TODO, revisit here to figure out why use this polynomial
-		let p = currency::cKMA;
-		let q = 100 * Balance::from(ExtrinsicBaseWeight::get());
-		smallvec![WeightToFeeCoefficient {
-			degree: 1,
+		smallvec!(WeightToFeeCoefficient {
```

Comment:
Can you use [] instead of ()? Like
```rust
smallvec![
...
]
```
I think we will use different polynomial the future.
And put what exact current polynomial here.